package th.fight.fit.mpybackoffice.exception;

public class UserNotAvailableException extends Exception {

	public UserNotAvailableException() {
		super();
		// TODO Auto-generated constructor stub
	}

	public UserNotAvailableException(String message, Throwable cause, boolean enableSuppression,
			boolean writableStackTrace) {
		super(message, cause, enableSuppression, writableStackTrace);
		// TODO Auto-generated constructor stub
	}

	public UserNotAvailableException(String message, Throwable cause) {
		super(message, cause);
		// TODO Auto-generated constructor stub
	}

	public UserNotAvailableException(String message) {
		super(message);
		// TODO Auto-generated constructor stub
	}

	public UserNotAvailableException(Throwable cause) {
		super(cause);
		// TODO Auto-generated constructor stub
	}

}
